apps/aecore/lib/aecore/miner/worker.ex

```diff
@@ -263,9 +265,21 @@ defmodule Aecore.Miner.Worker do
     end
   end
 
-  defp filter_transactions_by_fee_and_ttl(txs, block_height) do
-    Enum.filter(txs, fn tx ->
-      Pool.is_minimum_fee_met?(tx, :miner, block_height) &&
+  defp filter_transactions_by_fee_and_ttl(txs, %{oracles: oracles}, block_height) do
```

Comment:
we can't prefilter transactions for mining with state. We have to consider the intermediary state tree that is build while adding the transactions to the candidate block for fee validation. (#634 not resolved)